gl.cpp

```diff
@@ -0,0 +1,164 @@
+#define SDL_MAIN_HANDLED
+#include "SDL2/SDL.h"
+#include "gl.h"
+
+// To compile: g++ fuzzy.cpp -lopengl32 -lglew32 -lSDL2 -lmingw32
+int main(int argc, char **argv)
```

Comment:
Maintenance tip: create a drawTeapot function so this isn't all in main and another draw function can be called in the future (ex. drawDragon, drawGame...)